x-pack/plugins/canvas/public/components/arg_form/arg_template_form.tsx

```diff
@@ -0,0 +1,98 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React, { useState, useEffect, useCallback, useRef } from 'react';
+import PropTypes from 'prop-types';
+import usePrevious from 'react-use/lib/usePrevious';
+import { RenderToDom } from '../render_to_dom';
+import { ExpressionFormHandlers } from '../../../common/lib/expression_form_handlers';
+
+interface Props {
```

Comment:
nit: what do you think about renaming `Props` -> `ArgTemplateFormProps`

Reply:
Done.